<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefWaveformDlg</class>
 <widget class="QWidget" name="DlgPrefWaveformDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Waveform Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="waveformTypeComboBox">
       <property name="toolTip">
        <string>Select from different types of displays for the waveform, which differ primarily in the level of detail shown in the waveform.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="frameRateLabel">
       <property name="text">
        <string>Frame rate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>frameRateSlider</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="frameRateSpinBox">
       <property name="toolTip">
        <string/>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QSlider" name="endOfTrackWarningTimeSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="openGlStatusLabel_2">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Average frame rate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="endOfTrackWarningTimeLabel">
       <property name="text">
        <string>End of track warning</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>endOfTrackWarningTimeSlider</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="waveformTypeLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string extracomment="The big waveform summary shows the waveform envelope of the track near the current playback position."/>
       </property>
       <property name="text">
        <string>Summary type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>waveformTypeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="openGlStatusLabel">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>OpenGl status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="waveformOverviewLabel">
       <property name="toolTip">
        <string extracomment="The smaller, zoomed-out version of the waveform shows the various markers within the track as well as the waveform envelope of the entire track."/>
       </property>
       <property name="text">
        <string>Overview type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="endOfTrackWarningTimeSpinBox">
       <property name="toolTip">
        <string>Highlight the waveforms when the last seconds of a track remains.</string>
       </property>
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="midVisualGain">
         <property name="toolTip">
          <string>Visual gain of the middle frequencies</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="visualGainLabel_5">
         <property name="text">
          <string>Low</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>allVisualGain</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="visualGainLabel_4">
         <property name="text">
          <string>Middle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>allVisualGain</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="visualGainLabel_2">
         <property name="text">
          <string>Global</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>allVisualGain</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="highVisualGain">
         <property name="toolTip">
          <string>Visual gain of the high frequencies</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="lowVisualGain">
         <property name="toolTip">
          <string>Visual gain of the low frequencies</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="visualGainLabel_3">
         <property name="text">
          <string>High</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>allVisualGain</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="allVisualGain">
         <property name="toolTip">
          <string>Global visual gain</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="normalizeOverviewCheckBox">
       <property name="text">
        <string>Normalize waveform overview</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QCheckBox" name="synchronizeZoomCheckBox">
       <property name="toolTip">
        <string>Synchronize zoom level across all waveform displays.</string>
       </property>
       <property name="text">
        <string>Synchronize zoom level across all waveforms</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSlider" name="frameRateSlider">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="visualGainLabel">
       <property name="text">
        <string>Visual gain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="buddy">
        <cstring>allVisualGain</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QComboBox" name="defaultZoomComboBox"/>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="waveformOverviewComboBox">
       <property name="toolTip">
        <string extracomment="The main waveforms">Select from different types of displays for the waveform, which differ primarily in the level of detail shown in the waveform.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="defaultZoomLabel">
       <property name="text">
        <string extracomment="Waveform zoom">Default zoom level</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>defaultZoomComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="frameRateAverage">
       <property name="toolTip">
        <string>Displays the actual frame rate.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="openGlStatusIcon">
       <property name="toolTip">
        <string>Displays which OpenGL version is supported by the current platform.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
